src/app/api/orders/route.tsx

```
@@ -0,0 +1,35 @@
+            return NextResponse.json(
+                { error: 'Invalid request body' },
+                { status: 400 }
+            );
+        }
+
+        // Mock response data
+        const response = {
+            orderId: body.deliveryInfoId,
+            totalPrice: body.deliveryInfoId,
+            orderedAt: new Date().toISOString(),
+            items: body.orderItems.map((item: any) => ({
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)